HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponent/InstallAppBottomViewModel.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  InstallAppBottomViewModel.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+
+import UIKit
+import GiniHealthAPILibrary
+
+protocol InstallAppBottomViewProtocol: AnyObject {
+    func didTapOnContinue()
+}
+
+final class InstallAppBottomViewModel {
+    
+    var giniHealthConfiguration = GiniHealthConfiguration.shared
```

Comment:
[Suggestion]: what about to use here only a button configuration and don't expose `GiniHealthConfiguration` from the model?

Reply:
We need these expose here for the `titleLabelFont` and for `moreInformationLabelFont` fonts required for UI

Reply:
I guess it might me `let` then :)